core/store/orm/orm.go

```diff
@@ -1357,27 +1347,22 @@ func (orm *ORM) LogConsumptionExists(lc *models.LogConsumption) (bool, error) {
 
 	var exists bool
 	err := orm.DB.DB().
-		QueryRow(query, lc.BlockHash, lc.LogIndex, lc.JobID).
+		QueryRow(query, blockHash, logIndex, jobID).
 		Scan(&exists)
 	if err != nil && err != sql.ErrNoRows {
 		return false, err
 	}
 	return exists, nil
 }
 
-// CreateLogConsumption creates a new LogConsumption record
-func (orm *ORM) CreateLogConsumption(lc *models.LogConsumption) error {
-	orm.MustEnsureAdvisoryLock()
-	return orm.DB.Create(lc).Error
-}
-
-// FindLogConsumer finds the consuming job of a particular LogConsumption record
-func (orm *ORM) FindLogConsumer(lc *models.LogConsumption) (models.JobSpec, error) {
+// MarkLogConsumed creates a new LogConsumption record
+func (orm *ORM) MarkLogConsumed(blockHash common.Hash, logIndex uint, jobID *models.ID) error {
```

Comment:
Is this simply a name change or is there more to it?

Reply:
Seems the diff didn't come out very clearly here.  With `MarkLogConsumed`, I collapsed 2 methods into one and renamed the remaining one.